src/producer/src/test/java/org/acme/ControllerResourceTest.java

```diff
@@ -0,0 +1,77 @@
+package org.acme;
+
+import com.github.tomakehurst.wiremock.WireMockServer;
+import com.github.tomakehurst.wiremock.client.WireMock;
+import com.github.tomakehurst.wiremock.stubbing.ServeEvent;
+import com.google.common.collect.Lists;
+import io.fabric8.openshift.client.server.mock.OpenShiftServer;
+import io.quarkus.test.junit.QuarkusTest;
+import io.quarkus.test.kubernetes.client.OpenShiftTestServer;
+import io.quarkus.test.kubernetes.client.WithOpenShiftTestServer;
+import io.restassured.RestAssured;
+import io.restassured.response.ValidatableResponse;
+import org.jboss.logging.Logger;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+import java.util.UUID;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.aResponse;
+import static com.github.tomakehurst.wiremock.client.WireMock.post;
+import static com.github.tomakehurst.wiremock.client.WireMock.postRequestedFor;
+import static com.github.tomakehurst.wiremock.client.WireMock.urlEqualTo;
+import static com.github.tomakehurst.wiremock.core.WireMockConfiguration.options;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.Matchers.hasSize;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@WithOpenShiftTestServer
+@QuarkusTest
+class ControllerResourceTest {
```

Comment:
Pls rename the class

Reply:
f52047663e507d7fa54be62d44167e6e5de5b8f7